File: burbuja_19-09-2025/Burbuja.cs
using System;

public class Burbuja   
{
    public static void Main(string[] args)
    {
        int[] arr = {64, 34, 25, 12, 22, 11, 90};
        Burbujafuncion(arr);
        Console.WriteLine("arreglo ordenado: " + string.Join(", ", arr));
    }
    public static void Burbujafuncion(int[] arr)
    {
        int n = arr.Length;
        int i, j, vaso;
        for (i = 0; i < n - 1; i++)
        {
            for (j = 0; j < n - i - 1; j++) 
            {
                if (arr[j] > arr[j + 1])
                {
                    vaso = arr[j];
                    arr[j] = arr[j + 1];
                    arr[j + 1] = vaso;
                }
            }
        }
    }
}

File: burbuja_19-09-2025/burbuja.js
function burbuja(arr) {
    let n = arr.length;
    let vaso;
    for (let i = 0; i < n - 1; i++) {
        for (let j = 0; j < n - i - 1; j++) {
            if (arr[j] > arr[j + 1]) {
                // Intercambia arr[j] y arr[j+1]
                vaso = arr[j];
                arr[j] = arr[j + 1];
                arr[j + 1] = vaso;
            }
        }
    }
    return arr;
}

arreglo = [64, 34, 25, 12, 22, 11, 90];
console.log("Arreglo original: " + arreglo);
console.log("Arreglo ordenado: " + burbuja(arreglo));